run

```diff
@@ -181,16 +166,11 @@ yarn_install () {
 
 clean_cache () {
   # Clean yarn cache volume
-  echo "Deleting yarn_cache Docker volume..."
-  docker volume rm yarn_cache 2>/dev/null || echo "No volume found"
-  if [ $LOCAL_YARN_INSTALLED = true ]; then
-    if [ $OPT_CACHE_CLEAN_FORCE = true ]; then
-      yarn cache clean
-    else
-      echo "#"
-      echo "# Yarn is locally installed, use --force to clean your local cache"
-      echo "#"
-    fi
+  if docker volume ls | grep -q yarn-cache; then
+    echo "Deleting cache volume:"
+    docker volume rm --force yarn-cache
```

Comment:
I am getting an error from --force but it looks like I am also getting it on the clean command. May be outside the scope of this.